R/sql.R

```diff
@@ -0,0 +1,167 @@
+#' @title Run SQL queries against DataFrame/LazyFrame data.
+#' @description Run SQL queries against DataFrame/LazyFrame data.
+#' @details Currently, only available when built with the `full` feature.
+#' See [polars_info()] for more information.
+#' @name SQLContext_class
+#' @keywords SQLContext
+#' @examplesIf pl$polars_info()$features$sql
+#' lf = pl$LazyFrame(a = 1:3, b = c("x", NA, "z"))
+#' res = pl$SQLContext(frame = lf)$execute(
+#'   "SELECT b, a*2 AS two_a FROM frame WHERE b IS NOT NULL"
+#' )
+#' res$collect()
+RPolarsSQLContext
+
+
+#' @title auto complete $-access into a polars object
+#' @description called by the interactive R session internally
+#' @param x RPolarsSQLContext
+#' @param pattern code-stump as string to auto-complete
+#' @return char vec
```

Comment:
```suggestion
#' @return Character vector
```
Can you clean the rest?

Reply:
ditto